lib/kennel/syncer.rb

```diff
@@ -42,7 +42,7 @@ def plan
 
     def confirm
       return false if noop?
-      return true if ENV["CI"] || !STDIN.tty?
+      return true if ENV["CI"] || !STDIN.tty? || !Kennel.err.tty?
```

Comment:
is checking only err for example good enough ? (just so all the checks are consistent)

Reply:
I suspect we don't need the ENV CI check, because I'd be surprised if the CI jobs run with a tty.

The err check is there because if you run the task with `< /dev/tty 2> err.log` then you'll never see the prompt.